src/rendertron.ts

```diff
@@ -86,6 +86,10 @@ export class Rendertron {
     if (!protocol.match(/^https?/)) {
       return true;
     }
+    console.log(parsedUrl);
+    if (parsedUrl.hostname && parsedUrl.hostname.match(/google.internal$/)) {
```

Comment:
should it be something like `^([^\.]+\.)*google\.internal$` so it doesn't match `fakegoogle.internal`? Or maybe it should be even broader: `\.internal$`?

Reply:
I'd err on the site of caution and just block /internal$/ at this point, I think.